grid-proxy/tools/db/generate.go

```diff
@@ -70,14 +83,32 @@ func generateTwins(db *sql.DB) error {
 			twin_id:      i,
 			grid_version: 3,
 		}
-		if _, err := db.Exec(insertQuery(&twin)); err != nil {
+
+		value, err := extractValuesFromInsertQuery(insertQuery(&twin))
+		if err != nil {
+			panic(err)
```

Comment:
the function returns the error, and the caller in generateData is acting on that value, so don't panic here, please refactor to return and the call can do whatever.